examples/acquiring_data/goes_xrs_example.py

```diff
@@ -30,10 +30,14 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
+from astropy.visualization import time_support
+
 from sunpy import timeseries as ts
 from sunpy.net import Fido
 from sunpy.net import attrs as a
 
+time_support()
```

Comment:
```suggestion
# Enable support for plotting astropy times
time_support()
```

I think we want to decide on how/where we include this call in our examples.

Reply:
Good point - what are your thoughts? seems like two options are
1. Under imports
2. Before first plot

I think I'd go with 2. personally, but am not too fussed.

Reply:
I think 2 as well.

Reply:
👍 I updated it to this and documented it in the dev docs